core/src/main/java/com/linecorp/armeria/common/HttpHeadersBuilder.java

```diff
@@ -55,6 +55,12 @@ public interface HttpHeadersBuilder extends HttpHeaderGetters {
      */
     HttpHeadersBuilder contentLength(long contentLength);
 
+    /**
+     * Sets whether the <a href="https://datatracker.ietf.org/doc/html/rfc2616#section-14.13">content-length</a>
+     * is unknown. If {@code true}, {@code content-length} header is not automatically updated.
+     */
+    HttpHeadersBuilder isContentLengthUnknown(boolean contentLengthUnknown);
```

Comment:
Should be OK without `is` prefix because it's a setter:
```suggestion
    HttpHeadersBuilder contentLengthUnknown(boolean contentLengthUnknown);
```
Also, if a user can always replace `contentLengthUnkown(false)` with `contentLength(actualLength)`, then we could even remove the boolean parameter, i.e. `contentLengthUnknown()` sets `true` to `isContentLengthUnknown`.